integration_test/itest/image.go

```diff
@@ -1,13 +1,54 @@
 package itest
 
-import "context"
+import (
+	"context"
+	"strings"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/dos"
+)
 
 type Image struct {
 	Name     string `json:"name"`
 	Tag      string `json:"tag,omitempty"`
 	Registry string `json:"registry,omitempty"`
 }
 
+func (img *Image) FQName() string {
+	nb := strings.Builder{}
+	if img.Registry != "" {
+		nb.WriteString(img.Registry)
+		nb.WriteByte('/')
+	}
+	nb.WriteString(img.Name)
+	if img.Tag != "" {
+		nb.WriteByte(':')
+		nb.WriteString(img.Tag)
+	}
```

Comment:
can we write `:latest` here if no tag? i just like the explicitness of it; sometimes you run into a repo that doesn't have a latest tag and it's confusing as hell to get a message like `dockerhub has no mysql image`